### ISSUE TITLE
Accessibility: ProjectViewer

### ISSUE THREAD

**Issue by alansemenov** [enonic/app-contentstudio#7720]
ProjectViewer div should be tabbable, have `role="button"` and `aria-haspopup="dialog"`. Enter should open the dialog.
